### PR TITLE
Address Dependency Vuln(s)

### PR DESCRIPTION
**Pull request by kingthorin**
Dependbot is currently listing one vuln.
npm audit was showing 4.

This is the result of `npm audit fix`

### CODE REVIEW

**Comment by kingthorin** [comment]
Verified working locally with docker: https://github.com/zaproxy/zaproxy-website?tab=readme-ov-file#docker

File: package-lock.json [hunk 1]
```diff
@@ -8332,9 +8332,9 @@
       }
     },
     "node_modules/hugo-bin": {
-      "version": "0.125.0",
-      "resolved": "https://registry.npmjs.org/hugo-bin/-/hugo-bin-0.125.0.tgz",
-      "integrity": "sha512-cTvHzQdK04KrBaW3jPdd/JAQXCOdXT1GiPHaMxe0YtLTVCV0ElgxkvXn6eQJUDl4QqM8+IlsFg/b2ofz6XS//A==",
+      "version": "0.125.1",
```
Comment (thc202) [inline package-lock.json:8335]: While not strictly needed would be good to keep the `package.json` in sync.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by thc202** [comment]
https://github.com/zaproxy/zaproxy-website/pull/2709#issuecomment-2206903798

You can also see that in the CI check (the screenshot is a good proof).

File: package-lock.json (hunk 1 above)
Comment (kingthorin) [reply package-lock.json:8335]: Oh I can fire another PR in. I just didn't know how to do that. I kinda figured npm woulda done whatever was needed.

Comment (thc202) [reply package-lock.json:8335]: Dependabot should end up doing that in the next update.

Comment (kingthorin) [reply package-lock.json:8335]: Okay that's fair, but any idea for next time?

Comment (thc202) [reply package-lock.json:8335]: I think an update before doing the audit fix.

**Pull request merged**